mandatory/3-playing-computer.js

```diff
@@ -7,12 +7,19 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
+      b is not declared
   2. Remove the line that throws the error.
+      commented out console.log(b);
   3. What is printed to the console?
+    2, 6, 4, 9, 6, 13, 8
   4. How many times is "f1" called?
+      2
   5. How many times is "f2" called?
+      3
   6. What value does the "a" parameter take in the first "f1" call?
+      7
   7. What is the value of the "a" outer variable when "f1" is called for the first time?
+      6
```

Comment:
This is 8. The initial value of 'a' is 6 and incremented by 1 with each iteration of the for loop. As the f1 loop is only called on the second iteration as the 'i' variable is odd at that point the 'variable' is incremented by 1 twice